README.md

```diff
@@ -69,15 +69,23 @@ Pass other parameters to jest after the `--` flag. For example,
 lwc-jest --debug -- --no-cache
 ```
 
+### Debugging in Visual Studio Code
+
+If you prefer to debug directly inside the Visual Studio Code IDE, follow these steps:
+- From the Visual Studio Code dropdowns, select Debug --> Add Configuration...
+- Replace the contents of generated `launch.json` with the [`Debug Jest Tests` configuration](https://jestjs.io/docs/en/troubleshooting#debugging-in-vs-code) provided by Jest
```

Comment:
When I click through to that link I don't see `Debug Jest Tests`. Would it be more clear to say:
```suggestion
- Replace the contents of the generated `launch.json` with the version from Jest's [_Debugging in VS Code_ documentation](https://jestjs.io/docs/en/troubleshooting#debugging-in-vs-code) provided by Jest
```

Reply:
"Debug Jest Tests" is the name of the launch.json configuration in that section. I can see how people may miss that though I'll update.